modules/custom/az_event/az_event.module

```diff
@@ -223,7 +223,7 @@ function az_event_form_views_exposed_form_alter(&$form, FormStateInterface $form
 function az_event_node_presave(EntityInterface $entity) {
   if ($entity->bundle() === 'az_event'
     && $entity instanceof NodeInterface
-    && !empty($entity->field_az_featured_image_as_thumb->value)) {
-    $entity->field_az_media_thumbnail_image->target_id = $entity->field_az_photos->target_id;
+    && !empty($entity->get('field_az_featured_image_as_thumb')->getValue())) {
```

Comment:
Line 226 seems to be returning `true` at all times since getValue() is returning an array with one item instead of returning the first element from that array, as `->value` did before. As a result, the thumbnail images for some events in the demo content are missing or incorrect. The code suggestion below seems to fix the issue. The same fix should be applied to line 79 in az_news.module.

```suggestion
    && !empty($entity->get('field_az_featured_image_as_thumb')->getValue()[0]['value'])) {
```

Reply:
would this work?
```
&& !empty($entity->get('field_az_featured_image_as_thumb')->get('value'))) {
```

Reply:
@trackleft I tested it and unfortunately it produces this error: 

> [error]  InvalidArgumentException: Unable to get a value with a non-numeric delta in a list. in Drupal\Core\TypedData\Plugin\DataType\ItemList->get() (line 101 of /app/web/core/lib/Drupal/Core/TypedData/Plugin/DataType/ItemList.php).